sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -94,7 +92,6 @@ def query_instrument(session, upstream_dependency, start_date, end_date, version
             models.FileCatalog.data_level == data_level,
             models.FileCatalog.version == version,
             models.FileCatalog.start_date >= datetime.strptime(start_date, "%Y%m%d"),
```

Comment:
Do we need to keep the end_date and put that into the start_date query here.
```suggestion
            models.FileCatalog.start_date >= datetime.strptime(start_date, "%Y%m%d"),
            models.FileCatalog.start_date <= datetime.strptime(end_date, "%Y%m%d"),
```
I'd be somewhat worried about a reprocessing finding lots of matches for files greater than some date without a restriction criteria on the upper end.

Reply:
Yes. I like the point of narrow but we don't have end_date now in the filename. should we construct end_date to be start_date + 24 hour?

Reply:
Maybe for now that is good, or just `start_date == ` instead? So we only get the start_date we explicitly want?

We will have to think later about what happens when I need all 3-months worth of daily files to create my 3-month map.

Reply:
ok. I will add TODO there and change comparison to `start_date == `